CHANGES/6910.misc

```diff
@@ -0,0 +1 @@
+Enable source-checkout of a plugin without also having a source checkout of pulpcore
```

Comment:
Can this be renamed to `CHANGES/6910.feature`. I think this is actually a feature.

Reply:
I'm not sure, as it only affects devel role

Reply:
I think of it like the plugin developers are a user persona, just not a typical user. We develop features for them, we fix bugs for them, etc.

Reply:
Maybe add in a statement like `For developers, ` at the beginning of the line?